astropy/table/tests/test_mixin.py

```diff
@@ -100,16 +101,32 @@ def test_io_ascii_write():
         if fmt['Write'] and '.fast_' not in fmt['Format']:
             out = StringIO()
             t.write(out, format=fmt['Format'])
-
```

Comment:
Please do not delete this line (or the one above).  Following PEP8 conventions there should be 2 lines between module functions.  Again, following the rule of minimizing diffs, avoid making changes (like whitespace) that are not strictly necessary for the patch you are doing.